server/.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 npm-debug.log
+uploads/*
+!uploads/.gitkeep 
```

Comment:
ok, I see, I changed and now it's working as expected without the ``` fs.mkdirSync(dir); ```. Multer is creating the folder according to the info in config.js